.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Automatically stale issue and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close Stale Issues
+        uses: actions/stale@v9.0.0
+        with:
+          days-before-pr-close: 30
+          days-before-issue-stale: 30
+          days-before-issue-close: 10
+          stale-issue-message: 'After 30 days of inactivity, this issue has been marked as stale. This issue will be auto-closed in 10 days. Comment, assign or remove the stale label to prevent this.'
+          stale-issue-label: stale
+          close-issue-label: staled
+          close-pr-label: staled
```

Comment:
```suggestion
          days-before-pr-stale: 30
          days-before-pr-close: 7
          days-before-issue-stale: 30
          days-before-issue-close: 7
          stale-pr-message: 'This pull request has been inactive for 30 days and will be closed in another week. If you want to give it another month, leave a comment, assign it to someone, or remove a `Stale` label.'
          stale-issue-message: 'This issue has been inactive for 30 days and will be closed in another week. If you want to give it another month, leave a comment, assign it to someone, or remove a `Stale` label.'
```

Reply:
- added `days-before-pr-stale` to make the behavior in-line with issues
- changed close to `7` days as it better aligns with people's behavior; its easier to read it as "ok, I have another week" than "hmm, when 10 days land exactly?"
- removed `close` labels, as its already possible to query for issues/prs that are closed+labeled with `Stale`
- removed `stale` label overrides, as it's already a default